src/run-graph.ts

```diff
@@ -52,6 +79,7 @@ export class RunGraph {
   private runList = new Set<string>()
   private resultMap = new Map<string, Result>()
   private throat: PromiseFnRunner = passThrough
+  private _running = new OutputWaitingList();
```

Comment:
Existing private members are not `_`-prefixed here

Reply:
👍